client/src/pages/DetailsPage/item-details.scss

```
@@ -0,0 +1,13 @@
+.sectionDivider {
+    border-top: 1px solid #000000;
+    margin: 20px 0;
+}
+
+.itemDetails {
+    article {
+        image {
+            width: 80vw;
```

`80vw` ist jetzt erstmal okay. Aber wenn Leute mit ultra-wide monitoren auf die Website gehen ist das bild RIESIG.
Versuche eine max width anzugeben:
```suggestion
            width: 80vw;
            max-width: 960px;
```

---

done! hab das erstmal provisorisch geschrieben gehabt. das styling kommt noch 👍